efs-lib/src/cmr/efs/connection.clj

```diff
@@ -0,0 +1,39 @@
+(ns cmr.efs.connection
+  "Contains functions for interacting with the EFS storage instance."
+  (:require
+   [clj-time.coerce :as cr]
+   [cmr.common.config :refer [defconfig]]
+   [cmr.common.log :refer (debug info warn error)]
+   [cmr.common.services.errors :as errors]
+   [cmr.common.services.health-helper :as hh]))
+
+(defn health-fn
+  "Returns the health status of the EFS instance."
+  []
+  )
+
+(defn health
+  "Returns the efs health with timeout handling."
+  []
+  ())
+
+(defn save-concept
```

Comment:
If a concept already exists does the save overwrite or leaves what is there?

Reply:
Saving will be based on current rules. If different revisions of a concept are kept, then saving to EFS will have to as well